src/StorageReader/ClaimController.ts

```diff
@@ -95,6 +95,26 @@ export class ClaimController {
         }
       )
 
+    const publishEntryFailureReason = async (
```

Comment:
I think this PR can be split. Adjusting current modules to publish failures does not require that anyone is listening to those published messages. The subscribes to those messages can be added after.

Reply:
Ok. I was questioning that. I'll split it up.